src/main/resources/i18n/messages.properties

```diff
@@ -247,9 +247,28 @@ ProjectsTable_create_new_project=Create New Project
 
 
 # Create Project Page
-projects.create.heading=Create a New Project
+server.CreateProject.error=There was an error creating the project.
+CreateProject.title=Create a New Project
+CreateProjectDetails.name=Project Name
+CreateProjectDetails.name-required=A name is required for every project
+CreateProjectDetails.name-characters=A project name can only have letters, numbers, spaces, _ and -.
+CreateProjectDetails.length=The ${label} must be at least ${min} characters long
+CreateProjectDetails.url=Expected format of url is https://wikiname.org
+
+CreateProjectSamples.locked=Some samples in the cart are either locked and cannot be copied to other projects.
+CreateProjectSamples.sampleName=Sample
```

Comment:
Shouldn't this be Sample Name?

Reply:
It could be.  If that is what you want.  Updated in 144944336ddaa525506120da47577570edb02723